tests/contracts/root_chain/test_tokens.py

```diff
@@ -1,28 +1,9 @@
-import pytest
-from eth_tester.exceptions import TransactionFailed
-from plasma_core.constants import NULL_ADDRESS
-
-
-def test_token_adding(token, plasma_framework):
-    assert not plasma_framework.hasToken(token.address)
-    plasma_framework.addToken(token.address)
-    assert plasma_framework.hasToken(token.address)
-    with pytest.raises(TransactionFailed):
-        plasma_framework.addToken(token.address)
-
-
-def test_token_adding_gas_cost(w3, plasma_framework):
+def test_exit_queue_adding_gas_cost(w3, plasma_framework):
```

Comment:
Removed other tests as it's not crucial and already tested in truffle End-To-End tests.